README.md

```diff
@@ -1,36 +1,52 @@
 # smart-faq
+
 Smart FAQ
 
-App demo: [https://kiwicom.github.io/smart-faq/](https://kiwicom.github.io/smart-faq/) 
+App demo: [https://kiwicom.github.io/smart-faq/](https://kiwicom.github.io/smart-faq/)
 
-## Instalation
+## Installation
 
 Clone the repo `git clone git@github.com:kiwicom/smart-faq.git`
 
 Install all the dependencies with `yarn`
 
-
 In order to be able to login one requires an `.env` file at the root of the codebase, with the following contents:
+
 ```javascript
-GRAPHQL_URI=XXXXXXXX
-KIWILOGIN_USER=XXXXXXXX
+GRAPHQL_URI = XXXXXXXX;
```

Comment:
why `;`? it's not needed

Reply:
sure, it was done by prettier format on save 🙂